syntax = "proto3";

message MsgResponse{
    bool error = 1;
    string message = 2;
}